apps/tlon-mobile/eas.json

```diff
@@ -23,13 +25,28 @@
     "production": {
       "extends": "base",
       "channel": "production",
+      "autoIncrement": true,
       "android": {
         "gradleCommand": ":app:bundleProductionRelease"
       },
       "ios": {
         "scheme": "Landscape"
       }
     },
+    "demo": {
+      "extends": "base",
+      "channel": "demo",
+      "distribution": "internal",
+      "autoIncrement": true,
+      "android": {
+        "gradleCommand": ":app:bundlePreviewRelease",
```

Comment:
@alecananian when I try to build the demo profile locally for Android, it warns me that this command will output the wrong artifact for internal distribution (`.aab` not `.apk`)

do you know what the gradle command should be to build for the preview bundle id but output the correct filetype here?

Reply:
Yeah I tried that, but Expo still warns. Haven't registered my Android device yet, so still need to try running it end to end and see if it ends up being fine.

Supporting iOS for these is more important anyhow, so I'll give that a shot after merging.